remo/base/static/base/css/fonts.less

```diff
@@ -2,26 +2,26 @@
     Fonts
 -------------------- */
 
-@mozilla-font-url: 'mozorg.cdn.mozilla.net/media/fonts/';
-
 @font-face {
-    font-family: 'Open Sans Light';
-    src: url('//@{mozilla-font-url}OpenSans-Light-webfont.eot');
-    src: url('//@{mozilla-font-url}OpenSans-Light-webfont.eot?#iefix') format('embedded-opentype'),
-         url('//@{mozilla-font-url}OpenSans-Light-webfont.woff') format('woff'),
-         url('//@{mozilla-font-url}OpenSans-Light-webfont.ttf') format('truetype'),
-         url('//@{mozilla-font-url}OpenSans-Light-webfont.svg#OpenSansLight') format('svg');
+    font-family: 'Fira Sans Light';
+    src: url('https://mozorg.cdn.mozilla.net/media/fonts/FiraSans-Light.eot');
+    src: url('https://mozorg.cdn.mozilla.net/media/fonts/FiraSans-Light.eot?#iefix') format('embedded-opentype'),
+         url('https://mozorg.cdn.mozilla.net/media/fonts/FiraSans-Light.woff2') format('woff2'),
+         url('https://mozorg.cdn.mozilla.net/media/fonts/FiraSans-Light.woff') format('woff'),
+         url('https://mozorg.cdn.mozilla.net/media/fonts/FiraSans-Light.ttf') format('truetype'),
+         url('https://mozorg.cdn.mozilla.net/media/fonts/FiraSans-Light.svg#OpenSansLight') format('svg');
     font-weight: normal;
     font-style: normal;
 }
 
 @font-face {
     font-family: 'Open Sans';
-    src: url('//@{mozilla-font-url}OpenSans-Regular-webfont.eot');
-    src: url('//@{mozilla-font-url}OpenSans-Regular-webfont.eot?#iefix') format('embedded-opentype'),
-         url('//@{mozilla-font-url}OpenSans-Regular-webfont.woff') format('woff'),
-         url('//@{mozilla-font-url}OpenSans-Regular-webfont.ttf') format('truetype'),
-         url('//@{mozilla-font-url}OpenSans-Regular-webfont.svg#OpenSansRegular') format('svg');
+    src: url('https://mozorg.cdn.mozilla.net/media/fonts/FiraSans-Regular.eot');
```

Comment:
Ι'd bring all those in this project. These paths in  bedrock are not guarantied to work.